datadog-service/0.15.0/README.md

```diff
@@ -0,0 +1,198 @@
+# datadog-service
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/datadog-service)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/datadog-service)](https://goreportcard.com/report/github.com/keptn-sandbox/datadog-service)
+
+This implements a datadog-service for Keptn. If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
```

Comment:
I'd like to see just a sentence or two about what DataDog is and why one wants to integrate it with Keptn.  Something like "This implements the `datadog-service` that integrates the Datadog observability platform with Keptn.  This enables you to use Datadog as the source for the Service Level Indicators ([SLIs](https://keptn.sh/docs/0.17.x/reference/files/sli/)) that are used for Keptn [Quality Gates](https://keptn.sh/docs/concepts/quality_gates/).

Reply:
Some more description would definitely be nice. I am going to steal your line. Thanks!

Reply:
Done